x/stakeibc/keeper/msg_server_claim_undelegated_tokens.go

```diff
@@ -121,6 +121,7 @@ func (k Keeper) GetRedemptionTransferMsg(ctx sdk.Context, userRedemptionRecord *
 	})
 
 	// Give claims a 10 minute timeout
+	// TODO: Remove reference to epoch time (make timeout relative to block time)
```

Comment:
does this TODO need to be executed?

Reply:
I don't think it should block the upgrade, but it's a change we should make at some point

Reply:
got it - we could use TODO [optimization] or TODO [cleanup] to make this more clear